### PR TITLE
add shape parameter for PlotBMRSummary 

### PR DESCRIPTION
**Pull request by QuayAu**
This is a PR for #1470 

This is still work in progress, as I need to add a unit test, that catches the problem that changing the shape parameter didn't have any effect. 
Is there any simple way to test if generated ggplots are the same?

### CODE REVIEW

File: R/plotBMRSummary.R [hunk 1]
```diff
@@ -53,8 +53,10 @@ plotBMRSummary = function(bmr, measure = NULL, trafo = "none", order.tsks = NULL
     levels(df$learner.id) = getBMRLearnerShortNames(bmr)
   }
 
-  p = ggplot(df, aes_string(x = meas.name, y = "task.id", col = "learner.id"))
+  p = ggplot(df, aes_string(x = meas.name, y = "task.id", col = "learner.id", shape = "learner.id"))
```
Comment (ja-thomas) [inline R/plotBMRSummary.R]: We don't really want to hardcode this right?

File: R/plotBMRSummary.R [hunk 2]
```diff
@@ -53,8 +53,10 @@ plotBMRSummary = function(bmr, measure = NULL, trafo = "none", order.tsks = NULL
...
   p = p + geom_point(size = pointsize, position = position_jitter(width = 0, height = jitter))
+  # set shape to standard shape 19
+  p = p + scale_shape_manual(values = rep(19, length(getBMRLearnerShortNames(bmr))))
```
Comment (ja-thomas) [inline R/plotBMRSummary.R]: Have a look at #1523, this might be problematic for wrapped learners

**Comment by berndbischl** [comment]
what is the status here? work in progress? or ready for review?

**Comment by QuayAu** [comment]
Work in progress, will have a look at it tomorrow

**Comment by larskotthoff** [comment]
What's the status here @QuayAu ?

**Comment by giuseppec** [comment]
Looks good to me. We internally set the shape to the default (using `scale_shape_manual`). This allows us to change the shape and replace the default from the outside using `scale_shape_manual`. This was not possible before this PR.

**Comment by giuseppec** [comment]
@QuayAu can you add an example as requested by @larskotthoff ?

**Pull request merged**